drivers/npmx/npmx_shell.c

```diff
@@ -2993,6 +3035,35 @@ static int adc_ntc_get(const struct shell *shell, size_t argc, char **argv,
 	return 0;
 }
 
+static bool adc_ntc_temperatures_recalculate(const struct shell *shell,
+					     temp_recalculate_data_t const *data)
+{
+	int16_t temperature;
+	uint32_t resistance;
+
+	npmx_error_t err_code = data->get_resistance_func(data->charger_instance, &resistance);
+	if (!check_error_code(shell, err_code)) {
+		shell_error(shell, "Error: unable to get old resistance value.");
+		return false;
+	}
+	if (!ntc_resistance_to_temperature(shell, data->ntc_config_old, resistance, &temperature)) {
+		shell_error(shell, "Error: unable convert old resistance to temperature.");
+		return false;
+	}
+	if (!ntc_temperature_to_resistance(shell, temperature, data->ntc_config_new, &resistance)) {
```

Comment:
Can the pointer to `data` be second parameter in `ntc_temperature_to_resistance()`?
It would be more readable.